### DISCUSSION TITLE
请教一下短链解析的方法🙏

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#434] [一般讨论]
为什么我请求的head里没有返回目标地址?
https://b23.tv/h5FD5JC
![image](https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/66856838/477c225c-4a4e-4633-90f7-bb6154397eb3)
https://github.com/Tsuk1ko/cq-picsearcher-bot/blob/900d7b04c7c938b6dedb4d475da097b9d1160e77/src/plugin/bilibili/index.mjs#L36-L49

**Comment by Tsuk1ko** [comment]
postman 帮你跳转了吧

你看下 postman 设置应该有个设置是否跟随重定向的

**Comment by Miuzarte** [comment]
> postman 帮你跳转了吧
> 
> 你看下 postman 设置应该有个设置是否跟随重定向的

解决了  感谢
